docs/csharp/getting-started/introduction-to-the-csharp-language-and-the-net-framework.md

```diff
@@ -1,13 +1,13 @@
 ---
-title: "Introduction to the C# Language and the .NET Framework"
+title: "Introduction to the C# Language and .NET"
```

Comment:
This article needs to be reviewed carefully to make sure the information provided in it are not .NET Framework-specific. Then it may be worth renaming the file and adding a redirection.

Reply:
@BillWagner This is an ancient intro to C#.  My inclination is to retire it and redirect to the new Tour of C#.  What do you think?

Reply:
Yes, that's on my plan for part of this release cycle (probably next month).